cmake-format-rapids-cmake.json

```diff
@@ -183,6 +183,16 @@
           "USE_PROPRIETARY_BINARY": 1
         }
       },
+      "rapids_cpm_nvtx3": {
+        "pargs": {
+          "nargs": 0
+        },
+        "kwargs": {
+          "BUILD_EXPORT_SET": 1,
+          "INSTALL_EXPORT_SET": 1,
+          "USE_PROPRIETARY_BINARY": 1
```

Comment:
@robertmaynard this is consistent with how the APIs are written, but I don't think this is right. We don't actually respect `USE_PROPRIETARY_BINARY` for nvtx3 unless I've missed something, it's just in the docstring (possibly copy-pasted?). Should we update the CMake function in this PR as well? I think we'd just want to remove [this line](https://github.com/rapidsai/rapids-cmake/blob/branch-24.08/rapids-cmake/cpm/nvtx3.cmake#L55).

Reply:
I wondered about that too. If it's truly not being used then I am in favor of taking it out.

Reply:
Yeah drop it, it should only be documented for nvcomp.